sdks/go/sdk.go

```diff
@@ -132,17 +140,22 @@ func (s *SDK) WatchGameServer(f GameServerCallback) error {
 	if err != nil {
 		return errors.Wrap(err, "could not watch gameserver")
 	}
-
+	log := func(gs *sdk.GameServer, msg string, err error) {
+		if time.Unix(gs.ObjectMeta.DeletionTimestamp, 0).IsZero() {
```

Comment:
```suggestion
		if gs.ObjectMeta.DeletionTimestamp.IsZero() {
```
This _should_ work 🤔 , here is an example - we do this a bunch of places.

https://github.com/googleforgames/agones/blob/80f74c4e0f257683cbe5318ae222368cf2442537/pkg/gameservers/controller.go#L428

More examples: https://github.com/search?q=repo%3Agoogleforgames%2Fagones%20IsZero&type=code

Reply:
Okay, big slice for humble 🥧 for me. I didn't realise this was the protobuf version, not the K8s API version 🤦🏻 

So, we can simplify this to:

```go
gs.ObjectMeta.DeletionTimestamp == 0
```

And that will make everything happy.

Sorry about that!